From 205f4c02addf58f1e6cf2705524f698d86159311 Mon Sep 17 00:00:00 2001
From: Xiaoliang Yang <xiaoliang.yang_1@nxp.com>
Date: Thu, 18 May 2023 18:16:20 +0800
Subject: [PATCH 4/4] header: ethtool: update ethtool_fp parameters

The preemption of ethtool added a new disabled parameter in Linux,
update it in ethtool.h on lldpd.

Signed-off-by: Xiaoliang Yang <xiaoliang.yang_1@nxp.com>
---
 include/linux/ethtool.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/ethtool.h b/include/linux/ethtool.h
index c148b36..b3a3d2c 100644
--- a/include/linux/ethtool.h
+++ b/include/linux/ethtool.h
@@ -343,6 +343,7 @@ struct ethtool_eee {
 
 struct ethtool_fp {
 	__u32   cmd;
+	__u8	disabled;
 	__u8    fp_supported;
 	__u8    fp_enabled;
 	__u8    fp_status;
-- 
2.25.1

